cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamFileJanitorService.java

```diff
@@ -67,7 +68,7 @@ public void run() {
           janitor.cleanAll();
           LOG.debug("Completed stream file cleanup.");
         } catch (Throwable e) {
-          LOG.warn("Failed to cleanup stream file: {}", e.getMessage());
+          LOG.warn("Failed to cleanup stream files.", e);
```

Comment:
I guess if its not happening frequently then ok to log the stack trace in warn 👍